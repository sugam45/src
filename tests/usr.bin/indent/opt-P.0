/* $NetBSD: opt-P.0,v 1.2 2021/03/07 10:12:18 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Mentioning another profile via -P has no effect since only a single
 * profile can be specified on the command line, and there is no 'include'
 * option.
 */
