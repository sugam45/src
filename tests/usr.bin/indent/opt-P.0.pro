/* $NetBSD: opt-P.0.pro,v 1.1 2021/03/06 17:56:33 rillig Exp $ */
/* $FreeBSD$ */

/*
 * It's syntactically possible to specify a profile file inside another
 * profile file.  Such a profile file is ignored since only a single
 * profile file is ever loaded.
 */

-P/nonexistent
