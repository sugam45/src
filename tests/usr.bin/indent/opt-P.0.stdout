/* $NetBSD: opt-P.0.stdout,v 1.3 2021/03/14 01:34:13 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Mentioning another profile via -P has no effect since only a single profile
 * can be specified on the command line, and there is no 'include' option.
 */
